kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -1023,6 +1023,90 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     message: 'Folders',
     context: "Tab label in the 'Browse channel' page that allows to navigate through its topics.",
   },
+
+  // TODO - move these into diff sections as we make this a full feature in 0.16
+  // Past Papers Project (12/2021) strings
+  scoreLabel: {
+    message: 'Score',
+    context:
+      "In the 'Reports' tab, coaches can see the score that each learner has obtained on a quiz.\n\nThis is represented as a percentage and indicates the questions they've answered correctly.",
+  },
+  timeSpentLabel: {
+    message: 'Time spent',
+    context: 'Indicates the time a learner has taken on a specific learning resource.',
+  },
+  practiceQuizReportTitle: {
+    message: 'Report for {quizTitle}',
```

Comment:
@radinamatic @sairina the context for this string reads `title of the quiz that report corresponds to`. To me, a string that reads: **"Report for <quiz title>"** relates to and describes **a report** for a specific quiz whose name will be automatically defined, rather than being about the quiz title. The main subject of the sentence is Report, not quiz. Am I reading this wrong? In case this is indeed about the Report, I'd recommend the context: **"Label for a report corresponding to a specific quiz".**